invenio_sipstore/models.py

```diff
@@ -62,6 +62,9 @@ class SIP(db.Model, Timestamp):
     content = db.Column(db.Text, nullable=False)
     """Text blob of the SIP content."""
 
+    status = db.Column(db.Text, nullable=False)
```

Comment:
Probably better to call it ``archived_at`` which is initial ``None`` and once the celery task is done, we set it to ``datetime.utcnow()``.